src/data/markdown/docs/02 javascript api/11 k6-execution/1-Execution.md

```diff
@@ -0,0 +1,137 @@
+| iterationInInstance | integer | The unique identifier of the current iteration across the current instance. |
+| iterationInTest     | integer | The unique identifier of the current iteration across the entire test. It is unique both in local, in cloud and in distributed or segmented executions. |
+
+### Instance
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationsInterrupted                | integer  | The amount of iterations interrupted.                                |
+| iterationsCompleted     | integer | The amount of iterations completed.  |
+| vusActive            | integer  | The amount of VUs active.                                                |
+| vusInitialized           | integer | The amount of VUs initialized from the start of the test.                                                                           |
+| currentTestRunDuration            | float   | The time passed from the start of the current test run in milliseconds.                    |
+
+### VU
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationInInstance | integer | The identifier of the iteration in the current instance.                                                                 |
+| iterationInScenario | integer | The identifier of the iteration in the current scenario.                  |
+| idInInstance        | integer | The identifier of the VU across the instance.                            |
+| idInTest            | integer | The global-unique identifier of the VU.                                  |
+
+## Examples and use cases
+### Get unqiue data once
+
+One of the common cases,  it's getting an unqiue and incremental index for accessing a dataset's item only once time per test.
+
+The `scenario.iterationInTest` field can be used for this case. Let's show an example:
+
+```
+import exec from "k6/execution";
+import {SharedArray} from "k6/data";
+
+var items = 10;
+var data = new SharedArray("mydata", function() {
+    var arr = new Array(items);
+	for (var i = 0; i< items; i++){
+		arr[i] = {"foo": "item" + i, "bar": "12345678"}
+	}
+	return arr;
+});
+
+export const options = {
+    scenarios: {
+        "use-all-the-data": {
+            executor: "shared-iterations",
+            vus: 100,
+            iterations: data.length, // here you can also multiply so it goes through it multiple times
+            maxDuration: "1h" // this will need to be big enough so that all the iterations can happen if that is what is wanted
+        }
+    }
+}
+
+export default function() {
+    var item = data[exec.scenario.iterationInTest];
+	console.log('my item:', item)
```

Comment:
There are tabs mixed with spaces in this example (same in the for loop above). Better use spaces everywhere for consistency.

```suggestion
    console.log('my item:', item)
```